internal/storage/storageutil/object_attrs.go

```diff
@@ -104,6 +104,27 @@ func ObjectAttrsToBucketObject(attrs *storage.ObjectAttrs) *gcs.Object {
 	}
 }
 
+func ObjectAttrsToBucketMinObject(attrs *storage.ObjectAttrs) *gcs.MinObject {
```

Comment:
not sure why we call it BucketMinObject.
If there is no reason, can you call the method as objectAttrsToMinObject

Reply:
Updated.